src/Compilers/CSharp/Portable/BoundTree/Statement.cs

```diff
@@ -9,6 +9,8 @@ partial class BoundStatement : IStatement
     {
         OperationKind IOperation.Kind => this.StatementKind;
 
+        bool IOperation.HasErrors => this.HasErrors;
```

Comment:
A more correct implementation is coming in the next iteration, which has different behavior for each class.